src/Core/Settings.cpp

```diff
@@ -5615,6 +5615,13 @@ Only available in ClickHouse Cloud. Number of background threads for speculative
 Use async and potentially multithreaded execution of materialized view query, can speedup views processing during INSERT, but also consume more memory.)", 0) \
     DECLARE(Int64, ignore_cold_parts_seconds, 0, R"(
 Only available in ClickHouse Cloud. Exclude new data parts from SELECT queries until they're either pre-warmed (see cache_populated_by_fetch) or this many seconds old. Only for Replicated-/SharedMergeTree.
+)", 0) \
+    DECLARE(Bool, short_circuit_function_evaluation_for_nulls, true, R"(
```

Comment:
As a developer, I find the description quite ambiguous. A normal user would not understand it at all. Kindly re-phrase and please add an example.